src/metaschema/shared-constraints/allowed-values-control-group-property-name.ent

```diff
@@ -1,3 +1,4 @@
 <enum xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0" value="label">A human-readable label for the parent context, which may be rendered in place of the actual identifier for some use cases.</enum>
 <enum xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0" value="sort-id">An alternative identifier, whose value is easily sortable among other such values in the document.</enum>
 <enum xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0" value="alt-identifier">An alternate or aliased identifier for the parent context.</enum>
+<enum xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0" value="method">its an assement method.</enum>
```

Comment:
I know I recommended to Paul because this is not ideal, but I am not sure where we would want this given discussion in the issue reported and how this prop is not constrained precisely in the assessment model. This property definition is in control common.

https://github.com/usnistgov/OSCAL/blob/v1.1.2/src/metaschema/oscal_control-common_metaschema.xml#L78-L81

I do not know where a more prudent place to put the constraint is, I think that is a part of the discussion between you and Brian, but if we do not do it here we have to duplicate a few times.

Reply:
I believe the side effect here is it makes this allowed on groups which is not correct.

Reply:
It's a shame these are defined in external constraints, because targeting them in multiple places would be easier, without the need to use this entity hack.

Reply:
I've updated this to exist on just the part assembly, it should be more "surgical now"